core/pref.c

```diff
@@ -92,6 +92,7 @@ struct preferences default_prefs = {
 	.auto_recalculate_thumbnails = true,
 	.extract_video_thumbnails = true,
 	.extract_video_thumbnails_position = 20,		// The first fifth seems like a reasonable place
+	.standard_level_grid = false,
```

Comment:
not sure I love the 'standard' in the name... isn't this a `three_m_based_grid` or something like that?
Variables have to start with a letter or underscore... because `3m_based` would be nicer, but not a valid name...

Reply:
Ok, "three_m_based_grid" it is.